<launch>
  <!--<node pkg="libuvc_camera" type="camera_node" name="front_camera">
    <param name="index" value="0" />
    <param name="width" value="640" />
    <param name="height" value="480" />
    <param name="vendor" value="0x0"/>
    <param name="product" value="0x0"/>
    <param name="serial" value=""/>
    <param name="video_mode" value="yuyv"/>
    <param name="frame_rate" value="15"/>
    <param name="camera_frame_id" value="front_camera" />
    <param name="auto_exposure" value="0" />
    <param name="exposure_absolute" value="-100" />
    <param name="gain" value="0" />
    <param name="brightness" value="0" />
    <param name="camera_name" value="front_camera" />
    <param name="timestamp_method" value="start"/>
    <param name="camera_info_url" value="file://$(find dev_package)/camera_info/front_camera.yaml" />
  </node>-->
  <node name="head_camera" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="head_camera" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="10" />
  </node>
  <node name="rect_test" pkg="dev_package" type="rect_test.py">
  </node>
  <node name="image_view" pkg="image_view" type="image_view">
    <remap from="image" to="rect_test/image_debug" />
    <param name="do_dynamic_scaling" value="false"/>
  </node>
  <node pkg="tf" type="static_transform_publisher" name="base_link_broadcaster" args="0 0 0 0 0 0 1 map base_link 100" />
</launch>
